src/loginServices.js

```diff
@@ -23,6 +24,10 @@ import {
   setUserServicePoints,
 } from './okapiActions';
 
+const styles = {
+  multipleErrorsUl: { textAlign: 'left', margin: '0' }
```

Comment:
Are you sure that we need to store styles in service? Maybe it should be moved to the error component?